src/hardware_trigger.cpp

```diff
@@ -183,6 +203,49 @@ HardwareTrigger::condition HardwareTrigger::digitalCondition(uint chnIdx)
 	return static_cast<condition>(it - lut_digital_trigg_cond.begin());
 }
 
+
+
+bool HardwareTrigger::hasExternalTriggerOut()
+{
+	auto ret = iio_channel_find_attr(m_logic_channels[1], "out_select");
+
+	return (ret==NULL) ? false : true;
+
+
+}
+bool HardwareTrigger::hasExternalTriggerIn()
+{
+	return true;
+}
+
+bool HardwareTrigger::hasCrossInstrumentTrigger()
+{
+	auto ret = iio_channel_find_attr(m_logic_channels[1], "logic_mode");
+	return true;
```

Comment:
What's the purpose of `ret` here, should it return `true`, or `(ret == NULL) ? false : true` like in `hasExternalTriggerOut()`?

Reply:
test code slipped through - fixed